warmup/bootstrapper.py

```diff
@@ -129,9 +154,12 @@ def bootstrap_steady_perf(steady_segments_all_pexecs, confidence_level=CONFIDENC
 
 
 if __name__ == '__main__':
-    import json
-    import sys
+    parser = argparse.ArgumentParser(description='Bootstrap data.')
```

Comment:
Is there any reason the argument is upper-case?

Reply:
Not sure I understand.

Reply:
I meant to comment on the following line, where we have 'HIGH' in all caps. Why not 'high'? Maybe you want the user to really understand the implications of deviating from the default?

Reply:
I'm just fitting in with the surrounding code (e.g. "debug"). I agree that it looks weird...